.chloggen/crosslinkErrSup.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. crosslink)
+component: crosslink
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Ignore ENOTTY error produced by zaps logger.sync() method
```

Comment:
The changelog entry now needs an update

Reply:
Good catch, thanks for the reminder. Updated.